testsupport/space.go

```diff
@@ -167,16 +163,16 @@ func VerifyResourcesProvisionedForSpaceWithTiers(t *testing.T, awaitilities wait
 	}
 
 	// get refs & checks
-	templateRefs, namespacesChecks, clusterResourcesChecks := tiers.GetRefsAndChecksForTiers(t, awaitilities.Host(), tierName, aliasTierNamespaces, aliasTierClusterResources)
+	templateRefs, checks := tiers.GetRefsAndChecksForTiers(t, hostAwait, tier)
 
 	// get NSTemplateSet
-	nsTemplateSet, err := targetCluster.WaitForNSTmplSet(spaceName, wait.UntilNSTemplateSetHasTier(tierName), wait.UntilNSTemplateSetHasConditions(Provisioned()))
+	nsTemplateSet, err := targetCluster.WaitForNSTmplSet(spaceName, wait.UntilNSTemplateSetHasTier(tier.Name), wait.UntilNSTemplateSetHasConditions(Provisioned()))
 	require.NoError(t, err)
 
 	// verify NSTemplateSet with namespace & cluster scoped resources
-	tiers.VerifyGivenNsTemplateSet(t, targetCluster, nsTemplateSet, namespacesChecks, clusterResourcesChecks, templateRefs)
+	tiers.VerifyNSTemplateSet(t, targetCluster, nsTemplateSet, checks, templateRefs)
 
-	if aliasTierNamespaces == "appstudio" {
+	if tier.Name == "appstudio" || tier.Annotations[tiers.BaseTierKey] == "appstudio" {
```

Comment:
there is another concern I have with using some additional annotations: It's not used in prod so we don't keep that in our minds that we would need look for it and check if it's set or not - that means that we could easily miss that in other checks plus it doesn't make any sense why we should check that for the first view.